README.md

```diff
@@ -294,6 +294,55 @@ Openstack Object Storage settings
 - **Project ID**: Openstack project ID
 - **Container**: Name of the storage container
 
+## Applying core patches
+
+This plugin requires various trackers to be backported to maintain the plugin functionality.
+
+| Moodle version   | Patches |
+|------------------|---------|
+| Moodle 3.1       | [MDL-46375](https://tracker.moodle.org/browse/MDL-46375), [MDL-58068](https://tracker.moodle.org/browse/MDL-58068) |
+| Moodle 2.7       | [MDL-46375](https://tracker.moodle.org/browse/MDL-46375), [MDL-58068](https://tracker.moodle.org/browse/MDL-58068), [MDL-49627](https://tracker.moodle.org/browse/MDL-49627) |
+
+#### Moodle 3.1:
```

Comment:
make these headings match the ranges above ie 3.1 - 3.x

Reply:
Done